src/resolvers/projectResolver.ts

```diff
@@ -449,20 +449,49 @@ export class ProjectResolver {
     );
     query = ProjectResolver.addUserReaction(query, connectedWalletUserId, user);
 
-    if (orderBy.field === 'traceCampaignId') {
-      // TODO: PRISMA will fix this, temporary fix inverting nulls.
-      const traceableDirection = {
-        ASC: 'NULLS FIRST',
-        DESC: 'NULLS LAST',
-      };
-      query.orderBy(
-        `project.${orderBy.field}`,
-        orderBy.direction,
-        // @ts-ignore
-        traceableDirection[orderBy.direction],
-      );
-    } else {
-      query.orderBy(`project.${orderBy.field}`, orderBy.direction);
+    switch (orderBy.field) {
+      case OrderField.Traceable: // TODO: PRISMA will fix this, temporary fix inverting nulls.
+        // const traceableDirection: {
```

Comment:
I guess we can save the snippets somewhere and remove the comments. Everything else looks ok.